### DISCUSSION TITLE
Addition of a wall function?

### DISCUSSION THREAD

**Discussion by Lyyce** [ProjectPhysX/FluidX3D#196] [Q&A] [answered]
Hello, I have been playing around with FluidX3D, in particular to compute drag and lift forces. 

It works well for low Reynolds number but struggles for highly turbulent flows.

The documentation mentions the possible addition of a wall function to allow for better accuracy at high Reynolds number, is there any update on this, or on local grid refinement?

**Comment by ProjectPhysX** [comment, marked as answer]
No update yet. I've been experimenting with friction slip bounaries but the DDF swapping fails for concave corner cells and I couldn't yet find a solution.
